src/cpp/session/modules/SessionHelp.cpp

```diff
@@ -776,6 +776,15 @@ void handleRdPreviewRequest(const http::Request& request,
 
    rCmd << filePath;
 
+   // instead of runCommand() 
+   // 
+   // call R code:
```

Comment:
I'm not sure how to do this, calling an `.rs.*` R function from the C++ side.

Reply:
For example:

https://github.com/rstudio/rstudio/blob/3bafffe479bbc3f762a9a29d0deb99ecdf9b348e/src/cpp/session/SessionModuleContext.cpp#L1221-L1228

The `RFunction` class can be used to call any R function on the search path, including the builtin RStudio `.rs.*` functions.